src/ui/components/Accordion.tsx

```diff
@@ -51,7 +51,7 @@ const Accordion = ({ items }: { items: AccordionItem[] }) => {
               </h2>
             </div>
             <div className={classNames(styles.content, { [styles.open]: !collapsed })}>
-              {component}
+              {!collapsed && component}
```

Comment:
One bummer about this is that state won't be maintained across component minimizing/maximizing. Maybe when we've improved the performance of that component we can turn this back on.